pkl-core/src/main/java/org/pkl/core/util/yaml/YamlEmitter.java

```diff
@@ -210,6 +212,11 @@ public final void emit(boolean value) {
     builder.append(value);
   }
 
+  public final void emit(byte[] value) {
+    builder.append("!!binary ");
+    builder.append(Base64.getEncoder().encodeToString(value));
```

Comment:
We should handle empty bytes; one way to do it is `!!binary ''`.

Perhaps a better approach is to append the tag, and then call `org.pkl.core.util.yaml.YamlEmitter#emit(java.lang.String, java.lang.StringBuilder, boolean)`. That will give you a well-formed string scalar.